.github/workflows/airflow.yml

```diff
@@ -7,18 +7,23 @@ on:
   push:
     branches: [main]
     paths:
-      - "lineapy/plugins/jinja_templates/*"
+      - "lineapy/plugins/jinja_templates/airflow/*"
+      - "lineapy/plugins/jinja_templates/task/*"
+      - "lineapy/plugins/jinja_templates/module/*"
       - "lineapy/plugins/airflow_pipeline_writer.py"
       - "lineapy/plugins/base_pipeline_writer.py"
```

Comment:
I think it would be difficult to mark all the files at this granularity. At some point we may want to have diff and dependency based test running so that only tests affected by changed files runs, but that is in the future. 

For now I think base_pipeline_writer is the lowest level of granularity that OSS contributors are expected to make changes to, so none of its dependencies are included.